react/components/Footer/global.css

```diff
@@ -1,5 +1,6 @@
 .vtex-footer {
   box-shadow: 4px -4px 8px 0 rgba(0, 0, 0, 0.2);
+  margin-top: 8px;
```

Comment:
Use tachyons `mt4`

Take a look at this, Lucy -> https://raw.githubusercontent.com/vtex/vtex-tachyons/master/tachyons.css

```md
<div className="vtex-footer mt4" ... />
```

Reply:
Done